### DISCUSSION TITLE
Encryption of payload during data transfer in NHCX

### DISCUSSION THREAD

**Discussion by VarunnSingla** [NHA-ABDM/nhcx#16] [Q&A] [answered]
How can I encrypt the payload during data transfer on NHCX  and is there a specific utility for this?

**Comment by vishi24** [comment, marked as answer]
Hi,

Please refer to the document https://hcxsbx.abdm.gov.in/images/d477c2951a917d480a22.docx.
 This contains the process and code snippets.
